Formula/virgil-cli-alpha.rb

```diff
@@ -0,0 +1,17 @@
+class VirgilCliAlpha < Formula
+  desc "CLI tool to manage Virgil applications"
+  homepage "https://github.com/VirgilSecurity/virgil-cli"
+  devel do
+    url "https://github.com/VirgilSecurity/virgil-cli/releases/download/v5.0.0-alpha4/virgil_5.0.0-alpha4_macOS_x86_64.tar.gz"
+    sha256 "3cc10f34560c2d4208b51dd3460049824524e1e68d6492d07bfe8aada8043513"
+  end
+
+  def install
+    bin.install "virgil"
+  end
+
+  test do
+    result = %x(`virgil -v`)
+    assert_equal "CLI version 5.0.0-alpha4, commit e97edf3e, built 2019-03-27T16:26:04Z\n", result
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.